docs/types.rst

```diff
@@ -270,7 +272,7 @@ Operator       Description
 ``x - y``      Subtraction
 ``-x``         Unary minus/Negation
 ``x * y``      Multiplication
-``x / y``      Division
+``x / y``      Decimal division
```

Comment:
```suggestion
``x / y``      Decimal division (10 decimal places precision)
```

Reply:
i don't think this is necessary, as the decimal type is already known to be 10 places